src/samples/Schema/Schema.cpp

```diff
@@ -63,14 +63,14 @@ int mainImpl(int argc, char** argv)
     }
     
     std::vector<const char*> channels;
-    names.pushback("Default");
```

Comment:
names does not exist, I've looked in the history and it was renamed to channels.